spiceaidocs/content/en/concepts/recommendations/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+type: docs
+title: "Recommendations"
+linkTitle: "Recommendations"
+weight: 13
+description: "Reference documentation for Spice.ai Recommendations"
+---
+
+- [API Reference]({{<ref "api#recommendations">}})
+
+Recommendations are the core of how Spice.ai can make apps intelligent. After a pod has been trained by specifying [Dataspaces]({{<ref "concepts/dataspaces">}}) which contain the data the AI should train on, the [Actions]({{<ref "reference/pod#actions">}}) that can be taken, the [Rewards]({{<ref "concepts/rewards">}}) for taking those actions, an app makes requests for intelligent recommendations on how it should behave based on new data.
+
+By default Spice.ai will recommend actions based on the latest data available, but this can be tweaked by asking for a recommendation at a specific time.
```

Comment:
```suggestion
By default, Spice.ai will recommend actions based on the last ingested observation, but this can be tweaked by asking for a recommendation at a specific time.
```